spec/sample_feeds/itunes.xml

```diff
@@ -20,14 +20,22 @@
 <itunes:email>john.doe@example.com</itunes:email>
 </itunes:owner>
 <itunes:image href="http://example.com/podcasts/everything/AllAboutEverything.jpg" />
-  <itunes:category text="Technology">
-    <itunes:category text="Gadgets"/>
-  </itunes:category>
-  <itunes:category text="TV &amp; Film"/>
-  <itunes:category text="Arts">
-    <itunes:category text="Design"/>
-    <itunes:category text="Food"/>
-  </itunes:category>
+<image>
+  <url>https://tenderlovemaking.com/images/header-logo-text-trimmed.png</url>
+  <title>Tender Lovemaking</title>
+  <link>http://tenderlovemaking.com</link>
+  <width>766</width>
+  <height>138</height>
+  <description>The act of making love, tenderly.</description>
+</image>
```

Comment:
Yeah, sorry I wasn't super clear. Just saying the example should probably be for the AllAboutEverything example.